project_event/models/activity_template.py

```diff
@@ -26,9 +26,31 @@ class ActivityTemplate(models.Model):
         'activity.category',
         string='Category',
     )
+    room_id = fields.Many2one(
```

Comment:
@rimbendhaou: temp_resp_id is not a good name for the field id. Is the store=True required for this temp_resp_id